Automattic/course
# Update theme description

This PR updates the theme description to something a bit more descriptive. It also updates the tags to improve discoverability, bumps the versions of some things, and adds a theme screenshot.

## Testing Instructions
- Go to _Appearance_ > _Themes_.
- Check that the Course theme displays an image.
- Hover over the Course theme and click _Theme Details_.
- Ensure the updated version, description and tags appear.
